test/integration/smart_answer_flows/check_uk_visa_test.rb

```diff
@@ -662,6 +662,24 @@ class CheckUkVisaTest < ActiveSupport::TestCase
         assert_current_node :outcome_transit_to_the_republic_of_ireland
       end
     end
+
+    context 'studying in the UK' do
+      setup do
+        add_response 'study'
+      end
+      context '6 months or less' do
+        should "takes you to outcome_study_m" do
+          add_response "six_months_or_less"
+          assert_current_node :outcome_study_m
+        end
+      end
+      context 'more than 6 months' do
+        should "takes you to outcome_study_y" do
+          add_response "longer_than_six_months"
+          assert_current_node :outcome_study_y
```

Comment:
Remarking on the commit message: 
```
If 6 months or less -> Outcome 3 Study M
If 6 months or more -> Outcome 3 Study Y
```
should say 
```
If 6 months or less -> Outcome 3 Study M
If 6 months or more -> Outcome 2 Study Y
```

Reply:
I've updated the commit message. 👍